Error: unable to resolve dependency tree

```
$ npm i axios
$ npm i @vespaiach/axios-fetch-adapter

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: my-project@0.0.1
npm ERR! Found: axios@0.26.1
npm ERR! node_modules/axios
npm ERR!   axios@"^0.26.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer axios@"^0.21.1" from @vespaiach/axios-fetch-adapter@0.2.0
npm ERR! node_modules/@vespaiach/axios-fetch-adapter
npm ERR!   @vespaiach/axios-fetch-adapter@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Is this an issue on my end or does this package need to update its `axios` dependency?

--- a/package.json
+++ b/package.json
@@ -31,6 +31,6 @@
         "vite": "^2.8.0"
     },
     "peerDependencies": {
-        "axios": "^0.21.1"
+        "axios": "^0.26.1"
     }
 }

